nexus/pkg/server/sender.go

```diff
@@ -26,7 +27,7 @@ const (
 )
 
 type ResumeState struct {
-	FileStreamOffset map[chunkFile]int
+	FileStreamOffset fs.FileStreamOffsetMap
 	Error            service.ErrorInfo
```

Comment:
i though i got rid of the Error field in the resumeState (i don't think it is needed anymore)

Reply:
i was leaving it for now, but i think we can do another pass for some error stuff.